scripts/setup/al2/reinstall_cmake.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -eux
+
+CMAKE_VERSION="3.27.7"
+
+# Remove other versions of CMake
+sudo yum -y remove cmake
```

Comment:
It's a bit unexpected for a setup script to delete an existing version of a program. Can you clarify why this is needed?

Reply:
Tested that this works, resolving.